src/api/models/Business.ts

```diff
@@ -59,6 +60,11 @@ Business.init(
         address: {
             type: DataTypes.STRING,
             allowNull: false
+        },
+        email: {
+            type: DataTypes.STRING,
+            unique: true,
+            allowNull: false
```

Comment:
Is there an email type in zod? It would make more sense to check for a valid email since this is what we really want

Reply:
Yes there is. It's `string().email()`, I used it in the schema